manifests/base/files/gen-istio-cluster.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: istio-reader-service-account
-  namespace: istio-system
+  namespace: default
```

Comment:
Pretty sure this is not correct - the default install should still be in istio-system.